m2m-transfers/REPWRITERSPECS/BANNO.M2MTRANSFERS.V1.POW

```diff
@@ -0,0 +1,2952 @@
+ SLTRANSFERFOUND           = NUMBER
+ SVCXFEROUTFOUND           = NUMBER
+ SYMXCLIENTNUM             = NUMBER
+ SYMXINSTANCE              = NUMBER
+ SYSTEMMEMOMODE            = NUMBER
+ SYSTEMMEMOMODECHR         = CHARACTER ARRAY(1)
+ TARGETACCOUNT             = CHARACTER
+ TARGETACCOUNTVALIDATION   = NUMBER
+ TARGETACCTCLOSEDATE       = DATE
+ TARGETACCTTYPE            = CHARACTER
+ TARGETSLCLOSECODATE       = DATE
+ TARGETSLID                = CHARACTER
+ TARGETSLTRANLOC           = NUMBER
+ TARGETSLTYPE              = CHARACTER
+ TARGETVERIFY3             = CHARACTER
+ TESTCHR                   = CHARACTER
+ TESTLOOP                  = NUMBER
+ TESTNAME                  = CHARACTER
+ TESTNAMETYPE              = CHARACTER
+ THISACCOUNT               = CHARACTER
+ TMPAMTCHR                 = CHARACTER
+ TMPCHR                    = CHARACTER
+ TMPDATE                   = DATE
+ TMPEXPIREDATE             = DATE
+ TMPIDTYPE                 = NUMBER
+ TMPLOOP                   = NUMBER
+ TMPNUM                    = NUMBER
+ TMPSID                    = CHARACTER
+ TRANACCOUNTNAME           = CHARACTER ARRAY(99)
+ TRANCOUNT                 = NUMBER
+ TRANCOUNTMAX              = 99
+ TRANDAY1                  = NUMBER ARRAY(99)
+ TRANDAY2                  = NUMBER ARRAY(99)
+ TRANERROR                 = CHARACTER
+ TRANFREQALT               = 1
+ TRANFREQSTD               = 0
+ TRANFREQUENCY             = NUMBER ARRAY(99)
+ TRANLOC                   = NUMBER ARRAY(99)
+ TRANNEXTTRANDATE          = DATE ARRAY(99)
+ TRANREADONLYSTATUS        = CHARACTER ARRAY(99)
+ TRANRECIPACCT             = CHARACTER ARRAY(99)
+ TRANRECIPACCTID           = CHARACTER ARRAY(99)
+ TRANRECIPACCTTYPE         = CHARACTER ARRAY(99)
+ TRANRECIPEXPIREDATE       = DATE ARRAY(99)
+ TRANRECIPMBRID            = CHARACTER ARRAY(99)
+ TRANRECIPNAME             = CHARACTER ARRAY(99)
+ TRANRECIPNICKNAME         = CHARACTER ARRAY(99)
+ TRANRECIPTYPECHR          = CHARACTER ARRAY(5)
+ TRANSFERFREQCHR           = CHARACTER ARRAY(1,15)
+ TRANSFERFREQMAX           = 14
+ TRANSOURCEACCT            = CHARACTER ARRAY(99)
+ TRANSTARTDATE             = DATE ARRAY(99)
+ TRANTYPE                  = NUMBER
+ TRANTYPEIMMEDIATE         = 1
+ TRANTYPEREGULAR           = 0
+ TRANXFERAMOUNT            = MONEY ARRAY(99)
+ TRUECHR                   = "true"
+ TRUEFALSECHR              = CHARACTER ARRAY(1)
+ VALIDACCOUNT              = NUMBER
+ VALIDCHRINPUT1            = CHARACTER
+ VALIDPARAMS               = NUMBER
+ VALIDRANGEINPUT           = NUMBER
+ WARNINGFOUND              = NUMBER
+ WARNINGSACCT              = NUMBER ARRAY(999)
+
+ TMPLOCATOR=NUMBER
+ VALIDDATA=NUMBER
+ VALIDACCTNUM=NUMBER
+ TMPACCT=CHARACTER
+ NLTSCHR=CHARACTER
+ VALIDLIMITSEXCEEDED=NUMBER
+END [DEFINE]
+
+SETUP
+ [*  INCLUDE "BNO.INTERACTIVE.INIT"
+*]
+ INCLUDEPROGRAMINFO=TRUE
+ LASTMODDATE='04/20/21'
+ LASTMODTIME="07:45 MT"
+ PROGRAMUPDATENOTE1="WORK IN PROGRESS - NOT READY FOR PRODUCTION."
```

Comment:
Did you want to change this from NOT READY FOR PRODUCTION to something like ready for beta...?